docs/QUICK_START.md

```diff
@@ -0,0 +1,202 @@
+# mLRS Documentation: Quick Start Guide #
+
+([back to main page](../README.md))
+
+This page provides hints, suggestions and links to help get new users started with mLRS quickly.
+
+This guide covers mLRS in its core use case to provide a single radio link to carry both RC control and full bi-directional MAVLink Telemetry.  When we discuss configuration of your flight controller, we assume you are running ArduPilot.  We also assume you use OpenTX or EdgeTX and that you will connect a ground station and/or use the [Yaapu Telemetry App](CRSF.md).
+
+The steps here should be followed in order and when practical we provide a way to confirm a step was completed successfully.
+
+We expect that you are starting fresh and that all ArduPilot and mLRS parameters not covered here still have their default settings.
+
+Some notation and terminology:
+- We use "Tx module" and "receiver" to refer to a mLRS module in the Tx/transmitter role and Rx/receiver role. In the use case assumed here, the Tx module will be installed in a OpenTX or EdgeTX radio handset, and the receiver will be mounted on the vehicle and connected to the flight controller.
+- We use lower case "rx" and "tx" when referring to the receive and transmit pads or pins of a serial port connection.
+
+## 1. Choose and purchase a Tx module and a compatible receiver
+
+There are many supported choices of mLRS hardware, including off-the-shelf options and  many DIY designs.
```

Comment:
"... mix Tx module and receiver ..."

pl don't use Tx and Rx, use Tx module and receiver ...